packages/item-cc/src/item.controller.ts

```diff
@@ -155,15 +164,67 @@ export class ItemController extends ConvectorController {
   ) {
     //first check if item exists
     let item = await Item.getOne(itemId);
+
+    if (item.proposalAccepted) {
+      throw new Error('This proposal was already accepted, this cannot be changed anymore.');
```

Comment:
Does this mean an item can only be transferred once? Why is this needed? The solution before was to put the proposedOwner field to null after transfer

Reply:
No it means that if a proposal is accepted by the receiving party, he cannot unaccept it anymore (because the deal is then made, you cannot undo it anymore)